server/index.ts

```diff
@@ -20,6 +20,12 @@ import rateLimit from 'express-rate-limit'
 import getPort, { portNumbers } from 'get-port'
 import helmet from 'helmet'
 import morgan from 'morgan'
+import { type prometheus as prometheusType } from '#server/prometheus.server.ts'
+// @ts-ignore - this file may not exist if you haven't built yet, but it will
+// definitely exist by the time the dev or prod server actually runs.
+import { prometheus as _prometheus } from '#server-build/prometheus.server.js'
```

Comment:
I'm pretty sure you should be able to do this:

```
import { prometheus } from './prometheus.server.ts'
```

And that should get compiled properly. If not, I'm pretty sure there's something we can change in our build to make it work.